tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs

```diff
@@ -557,31 +558,6 @@ public void BuildsApplication_ByDynamicallyInstallingSDKs_IntoCustomDynamicInsta
                 var imageHelper = new ImageTestHelper();
 
                 // stretch
-                data.Add(
```

Comment:
It's a speculation from exit code 137.

![image](https://github.com/microsoft/Oryx/assets/86327553/c7275598-9f2c-4ed1-b12f-9a490a3ee4ed)

https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=8301382&view=logs&j=079e28c0-c837-57cb-f933-2a6580ab8338&t=2e7f4fc8-5913-529f-f546-06dafab89e5c&l=377

Reply:
Side note: the test in question `BuildsApplication_AfterInstallingSupportedSdk` succeeds locally
![image](https://github.com/microsoft/Oryx/assets/86327553/a8698ea7-8d6d-4eed-8837-12e83510d10c)

Reply:
happy to learn, thanks for sharing.

Reply:
In this case it wouldn't use Dynamic. It would use bookworm which we specify in our [parameter](https://github.com/microsoft/Oryx/pull/2098/files#diff-41b776f6f7b713085d0c1a0c018725d6ac56517c6d23cbf7889d2d603d6ef404R601).